### PR TITLE
Allow folder for controller specs to be configured 

### LINKED ISSUE #23: Auto-create file for controllers should create a request spec (or be configurable)
Hi,

As you may already know, controller specs are deprecated, from the [release notes](http://rspec.info/blog/2016/07/rspec-3-5-has-been-released/):

>For new Rails apps: we don't recommend adding the rails-controller-testing gem to your application. The official recommendation of the Rails team and the RSpec core team is to write request specs instead. Request specs allow you to focus on a single controller action, but unlike controller tests involve the router, the middleware stack, and both rack requests and responses. This adds realism to the test that you are writing, and helps avoid many of the issues that are common in controller specs. In Rails 5, request specs are significantly faster than either request or controller specs were in rails 4, thanks to the work by Eileen Uchitelle1 of the Rails Committer Team.

Thus, whenever we auto-create a controller spec via the extension's "create file" feature, instead of creating the file in `spec/controllers`, it should create it in `spec/requests`. Or at least, this should be configurable.

---

I'm not familiar with the codebase but let me know how I can help.

### PR DESCRIPTION
**Pull request by davidalejandroaguilar**
Closes https://github.com/thadeu/vscode-run-rspec-file/issues/23

# What

This PR:

* Allow folder for controller specs to be configured.

# Why

As you may already know, controller specs are deprecated, from the [release notes](http://rspec.info/blog/2016/07/rspec-3-5-has-been-released/):

>For new Rails apps: we don't recommend adding the rails-controller-testing gem to your application. The official recommendation of the Rails team and the RSpec core team is to write request specs instead. Request specs allow you to focus on a single controller action, but unlike controller tests involve the router, the middleware stack, and both rack requests and responses. This adds realism to the test that you are writing, and helps avoid many of the issues that are common in controller specs. In Rails 5, request specs are significantly faster than either request or controller specs were in rails 4, thanks to the work by Eileen Uchitelle1 of the Rails Committer Team.

### CODE REVIEW

**Comment by thadeu** [comment]
Hey @davidalejandroaguilar Thanks for this PR.

So, I totally agree with you, but I think that we need keep configuration to controllers directory by default, because in many cases, many developers will use controllers instead requests. 

But, I would like merge this PR.

You could change default folder do controllers and then if someone developer prefer, will change this.

What do you think? Do you agree with me?

File: package.json [hunk 1]
```diff
@@ -99,6 +99,11 @@
         "configuration": {
             "title": "vscode-run-rspec-file",
             "properties": {
+                "vscode-run-rspec-file.controller-spec-directory": {
+                    "type": "string",
+                    "default": "requests",
```
Comment (thadeu) [inline package.json:104]: "default": "controllers",

File: README.md [hunk 2]
```diff
@@ -77,6 +77,16 @@ The default is `bundle exec rspec --color`.
 
 You can also customize it per project by adding the same configuration to your project's `.vscode/settings.json`.
 
+### Custom folder for controller specs
+
+You should now write specs for your controllers as request specs ([source](http://rspec.info/blog/2016/07/rspec-3-5-has-been-released/)). The default thus is to create controller specs at `spec/requests`, but this can be changed via:
+
+```json
+{
+  "vscode-run-rspec-file.controller-spec-directory": "controllers"
```
Comment (thadeu) [inline README.md]:
```suggestion
  "vscode-run-rspec-file.controller-spec-directory": "requests"
```

File: package.json (hunk 1 above)
Comment (thadeu) [inline package.json]:
```suggestion
                    "default": "controllers",
```

**Review by thadeu — COMMENTED** [review]
@davidalejandroaguilar What do you think of the suggestions?

**Comment by davidalejandroaguilar** [comment]
Hello @thadeu, sorry for the huge delay. I thank you for considering the PR and agree with you, so I left the `controllers` directory as default and just made it configurable to `requests`.

**Comment by thadeu** [comment]
@davidalejandroaguilar Great, thanks for you PR.

LGTM.

**Comment by thadeu** [comment]
I'll compile branch and submit new version to the store.

**Pull request merged**